File: OstjApp/OstjApi/Models/Resume.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace OstjApi.Models
{
    [Table("Resumes")]
    public class Resume
    {
        public int Id { get; set; }
        public required int PersonId { get; set; }
        public required string Content { get; set; }
    }
}

File: resume-matching/src/main/java/com/ostj/dataentity/Resume.java
package com.ostj.dataentity;

public class Resume
{
    public int Id = -1;
    public int PersonId = -1;
    public String Content = "";

    public String toString(){
        return String.format("Id=%d, PersonId=%s, Content=%s", Id, PersonId, Content );
    }
}